cds_wan_point_to_point_load_balancer/README.md

```diff
@@ -0,0 +1,53 @@
+## Peer-to-Peer Communication with Participants behind Cone NATs Using Cloud Discovery Service using a Load Balancer
+
+### Problem
+
+You want 2 DDS applications outside a K8s cluster to communicate with each other, but none of their public IP addresses is known. Plus, you would like to use a Load Balancer that will also give you a DNS address. This is a similar use case of [this](../cds_wan_point_to_point_node_port).
+
+### Solution
+
+To solve this, **Cloud Discovery Service** is used to resolve the public IP address of the 2 applications. The **Load Balancer** will expose CDS on a static port on the known public IP address of the cluster, with a DNS address.
```

Comment:
I think we could say like the public IP address provided by the external load balancer of a cloud provider.